src/main/java/com/conveyal/r5/analyst/cluster/GridResultAssembler.java

```diff
@@ -0,0 +1,201 @@
+package com.conveyal.r5.analyst.cluster;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClient;
+import com.amazonaws.services.sqs.model.Message;
+import com.google.common.io.ByteStreams;
+import com.google.common.io.LittleEndianDataInputStream;
+import com.google.common.io.LittleEndianDataOutputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.RandomAccessFile;
+import java.util.Base64;
+import java.util.zip.GZIPOutputStream;
+
+/**
+ * Assemble the results of GridComputer into AccessGrids.
+ *
```

Comment:
Rather than just naming the input and output classes (which are visible from function signatures), we should describe the data transform that occurs, and when it happens. "During distributed computation of access to gridded destinations, workers place raw accessibility results for single origins onto an Amazon SQS queue. These results contain one accessibility measurement per iteration (departure minutes and Monte Carlo draws) per origin grid cell. This class pulls results off the queue as they become available, and assembles them into a single large file containing a delta-coded version of the same data for all origin points."

AccessGrids should probably be abstracted out into another class as well, with OO-style "message" methods for reading and writing results for a given origin.